test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1478,6 +1488,17 @@ public Builder columns(int columns) {
             return this;
         }
 
+        /**
+         * Set the HyperlinkListener of links inside the instructions pane.
+         *
+         * @param hyperlinkListener the listener
+         * @return this builder
+         */
+        public Builder addHyperlinkListener(HyperlinkListener hyperlinkListener) {
```

Comment:
```suggestion
        public Builder hyperlinkListener(HyperlinkListener hyperlinkListener) {
```
For consistency with other methods.